### ISSUE TITLE
Logged-in offline players' names change when new offline players log in, and players get mixed up in the Bukkit server.

### ISSUE THREAD

**Issue by larsr** [Thulinma/MixedModeAuth#4]
I am using the modded bukkit server in offline mode.

First I log in with one offline player as "Alice". Then I log in with another offline player as "Bob".

Then I make "Alice" say (using 't') "Hello". On the console it is printed "<Bob> Hello" but it should say "<Alice> Hello"!

When I list the users it says there are two users called "Bob".

So it seems that somehow the users are mixed up.  The player that loged in as Player
seem to get assigned to the same user, but the ones with a premium account seem to be
seen as different users.

There are other problems too which I think are related to this.
I am not sure that this is happening 100% of the time, but it happens very frequently.

**Comment by Thulinma** [comment]
Hmm... Interesting. This is the first time I've heard of this behaviour. Can you try and figure out what exactly causes this? Is it always with multiple offline players? Is there something you can do that causes the bug 100% of the time? Does it ever happen to online players?

Meanwhile, I'll look into it myself too and will see if I can figure out the cause...

**Comment by larsr** [comment]
Sorry, I closed the issue by mistake!   I also edited the post to show the brackets around <Alice> etc.

**Comment by Thulinma** [comment]
Yeah, accidental closing happens a lot on github, heh. I've done it myself a few times too. They should change those buttons, I think. :-)

**Comment by beaucarnes** [comment]
I am having this exact same issue. I am running in offline mode and the server seems to only be aware of the last person to log in. All players can play but the server only remembers what players do if they are either the last player to log in or near the last player to log in. If you are not near the last player to log in, when you place a block it will disappear in a few seconds and other players (that are also not the last player to log in) cannot see the blocks placed. Are there any similar mods that I can use since this one is not working for me? Or is there a fix for this issue?

**Comment by Thulinma** [comment]
I plan to fix this problem the best I can as soon as I make the necessary updates for MC 1.2.
I am not aware of any other mods/plugins that can do this, but there may be some that I haven't heard of.
